cli/src/main/scala/Main.scala

```diff
@@ -45,9 +64,14 @@ object Main {
           ()
         }
       } else {
+        val linker = WebAssemblyLinkerImpl.linker(linkerConfig)
+        val outputDir = "./target/sample/"
+        createDir(outputDir)
+        val output = NodeOutputDirectory(outputDir)
```

Comment:
This `else` branch is no longer needed. When we run `sample/fastLinkJS`, it's going to be handled by `WasmLInkerPlugin`.
Maybe we can remove this branch? I was a bit confused why `sample/fastLinkJS` doesn't generate output under `./target/sample`.

Reply:
Good point. I removed it.